examples/tutorials/doc_examples/examples/delegate.py

```diff
@@ -26,28 +26,28 @@ class Child(HasTraits):
 sally = Child(first_name='Sally', father=tony, mother=alice)
 
 # Child delegates its 'last_name' to its 'father' object's 'last_name'
-print sally.last_name
+print(sally.last_name)
 # Output: Jones
 
 # Assign an explicit value to the child's 'last_name'
 sally.last_name = 'Cooper'
-print tony.last_name
+print(tony.last_name)
 #Output: Cooper
 
 # Validation is still controlled by the father's 'last_name' trait
-print 'Attempting to assign a Parent object to a Str trait...\n'
+print('Attempting to assign a Parent object to a Str trait...\n')
 try:
     sally.last_name = sally.mother  # ERR: string expected
-except TraitError, c:
-    print 'TraitError: ', c
+except TraitError as c:
+    print('TraitError: ', c)
```

Comment:
Another case where either formatting or a `__future__` import should be used. I'll stop commenting on these at this point ...